docs/usage.md

```diff
@@ -125,6 +126,20 @@ Expected (uncompressed) database files for each tool are as follows:
   - `database100mers.kraken`
   - `database150mers.kmer_distrib`
   - `database150mers.kraken`
+- **KrakenUniq** output of `krakenuniq-build` command(s) A directory containing:
+  - `opts.k2d`
+  - `hash.k2d`
+  - `taxo.k2d`
+  - `database.idx`
+  - `database.kdb.counts`
+  - `database50mers.kmer_distrib`
+  - `database75mers.kmer_distrib`
+  - `database100mers.kmer_distrib`
+  - `database150mers.kmer_distrib`
+  - `database200mers.kmer_distrib`
+  - `database300mers.kmer_distrib`
+  - `seqid2taxid.map`
```

Comment:
```suggestion
```

Because of: https://github.com/fbreitwieser/krakenuniq/blob/3efe086be6ea351624c1e946436a9b422864b8f8/scripts/build_db.sh#L351